node/node.go

```diff
@@ -988,6 +993,7 @@ func (node *AlgorandFullNode) loadParticipationKeys() error {
 				return err
 			}
 		}
+		part.Close()
```

Comment:
this is probably not a right place since there are lots of `return` statement within the if block above.
I suggest to wrap the if block in a function and defer closing like this:
```
if err != nil {
  func() {
    defer part.Close()
    handle.Close()
    ...

  }()
}
```

Reply:
part is used when deleting part keys. So, closing here is not right.